### DISCUSSION TITLE
Openwrt - change makefile for building with support for raspberry pi camera

### DISCUSSION THREAD

**Discussion by DocSneider** [bluenviron/mediamtx#2203] [Questions]
Hello,

I successfully built the image for a raspberry pi Zero W, but I want to use the onboard camera interface of the raspi.

The problem is that with the standard openwrt-makefile provided, the support for the raspi-camera is not enabled.

The following part would somehow have to be included into the makefile, but I was not able to do so 😬  (I do not have that much experience with using makefiles)
```bash
cd internal/rpicamera/exe
make
cd ../../../
go build -tags rpicamera .
```

Is there anyone who could give me an advice on how to do this?

**Comment by aler9** [comment]
Hello, this server uses libcamera to interact with the Raspberry Pi Camera, but libcamera is only available for Raspberry Pi OS, not OpenWRT. If you're interested in the OpenWRT + RPI Camera combination, ask libcamera maintainers to add support for it.

[a comment by an automated account is not shown]
